### DISCUSSION TITLE
Καλώς ήρθες

### DISCUSSION THREAD

**Discussion by sonikf** [DoliGreece/DoliGreeceDevs#1] [Announcements]
Θέλεις να γίνεις μέλος και να συμμετέχεις στην προσπάθεια μας;

Άφησε εδώ το github username σου και σύντομα θα σου αποσταλεί πρόσκληση(invitation).

**Comment by SavvasMohito** [comment]
Καλησπέρα, θα ήθελα να βοηθήσω

**Reply by sonikf** [reply]
Γεια σου @SavvasMohito 
Ευχαριστώ πολύ!
Αν θες να συζητήσουμε, να μου πεις που μπορείς να συνεισφέρεις και  να σου εξηγήσω σε ποια φάση είμαστε, στείλε ενα pm στο https://www.dolibarr.org/forum/

**Comment by sapostolakis81-afk** [comment]
Καλησπέρα σας, 

παρακαλώ για την πρόσκληση το forum DoliGreece.
sapostolakis81-afk

**Reply by sonikf** [reply]
Γεια σου στείλε ένα pm @sonikf στο [international forum](https://www.dolibarr.org/forum/) με το τηλέφωνο σου αν θες.
